coco/request_forwarder.py

```diff
@@ -19,11 +19,12 @@
 class Forward:
     """Keep data about a forward to another endpoint."""
 
-    def __init__(self, name, group=None, request=None, check=None):
+    def __init__(self, name, group=None, request=None, check=None, timeout=None):
```

Comment:
Can you add in-code documentation on the expected unit for timeout? I.e. is it in `seconds`?